assets/javascript/index.js

```diff
@@ -0,0 +1,23 @@
+/* globals $ */
```

Comment:
I grabbed this snippet of code from the npm website, I think you're right that this line can go away.